src/test/java/com/napier/sem/DAO_IntegrationTests.java

```diff
@@ -75,36 +75,39 @@ void allCountriesIn_arrayIsEmptyWhenAreaNameIsInvalid()
         assertEquals(0, countries.size());
     }
 
-    /**
-     * Integration tests covering the DAO.topNCountriesIn method
-     */
-    // Tests that the number of results is less than or equal to integer n
+    // Tests that the results are listed in descending order.
     @Test
-    void topNCountriesIn_arrayIsCorrectSize()
+    void allCountriesIn_countriesPopulationIsInDescendingOrder()
     {
         // given
-        int n = 17;
+        String areaName = "Asia";
 
         // when
-        ArrayList<Record> countries = dao.topNCountriesIn(App.WORLD, "", n);
+        ArrayList<Record> countries = dao.allCountriesIn(App.CONTINENT, areaName);
 
         // then
-        assertTrue(countries.size() <= n);
+        int n = countries.size();
+        for (int x = 0; x < n-1; x++)
```

Comment:
it's most common to have the variable in a for loop be `i` for index. it's also what's used in mathematics. if you have a nested for loop it's most common to use `i`, then `j` e.g. 
```
for (int i = 1; i < 6; i++) {
    for (int j = 1; j < 6; j++) {
        print(i * j) // 1, 2, 3, 4, 5, 2, 4, 6, 8, 10, 3, 6, 9 ...
    }
}
```